lasagne/layers/normalization.py

```diff
@@ -101,3 +103,58 @@ def get_output_for(self, input, **kwargs):
             scale += self.alpha * input_sqr[:, i:i+ch, :, :]
         scale = scale ** self.beta
         return input / scale
+
+
+def standardize(layer, offset, scale, shared_axes):
+    """
+    Convenience function for standardizing inputs by applying a fixed offset
+    and scale.  This is usually useful when you want the input to your network
+    to, say, have zero mean and unit standard deviation over the feature
+    dimensions.  This layer allows you to compute those statistics over your
+    training set and include them as part of your network.  The statistics are
```

Comment:
"This layer allows you to compute those statistics" -- sounds as if the layer would do this. Can you rephrase such that it's clear that the layer only allows you to store the normalization offsets/factors within the network, but does not help you with computing them?